dist/ramda.js

```diff
@@ -6200,10 +6198,6 @@
     };
 
     var R = {
-        F: F,
-        I: I,
-        T: T,
-        __: __,
```

Comment:
I could have been on Windows last night; it was late at night and I was in the process of moving things from my old to my new machine, doing some work on both.  I might have committed from Windows, although if so from a POSIX emulator, _mingw_..  **But**, `Array.prototype.sort` should not be platform-dependent.  I haven't looked.  Is there a sort in there or is it based on the file system ordering, which could well be platform-dependent?